integrations/postgresql/sensu-integration.yaml

```diff
@@ -0,0 +1,115 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: postgresql
+  name: postgresql-health-check
+spec:
+  class: enterprise # community, supported, enterprise, or partner
+  provider: monitoring
+  display_name: "Sensu Enterprise datastore monitoring"
```

Comment:
Display_name is the default attribute that the web_ui search bar with search if you just type in a search string.  This is probably NOT the display_name string you want as it will never match for Postgres